Sources/Commands/Error.swift

```diff
@@ -30,7 +30,7 @@ extension Error: CustomStringConvertible {
         case .invalidToolchain(let problem):
             return problem
         case .rootManifestFileNotFound:
-            return "root manifest not found"
+            return "Could not find \(Manifest.filename) in this directory or any of it's parent directories."
```

Comment:
Instead of just "this directory", would it be better to print out the directory's path here?

If this error occurs to a dependency, the user might not know what "this directory" is, since likely the search for the dependency's manifest doesn't start from where the root package's does.

Reply:
There is already a separate error for this (which looks like it also is slightly broken):

> /private/tmp/test/dep2 @ main: error: /Package.swift has no Package.swift manifest in /private/tmp/test/dep2

Reply:
Nit: "it's" -> "its"
```suggestion
            return "Could not find \(Manifest.filename) in this directory or any of its parent directories."
```